### ISSUE TITLE
 No playable streams found on this URL - youtube live

### ISSUE THREAD

**Issue by wiemar7** [streamlink/streamlink#1547]
*Thanks for reporting an issue!*
*Please read the contribution guidelines first! (see the link above)*
*Also check the list of known issues before reporting an issue!*

*Feel free to use the following template. Be as detailed as possible.*
*Please see the text preview to avoid unnecessary formatting errors.*
*Don't forget to remove this text before submitting.*

----

### Checklist

- [ ] This is a bug report.
- [ ] This is a feature request.
- [x ] This is a plugin (improvement) request.
- [ ] I have read the contribution guidelines.

### Description
i can't download live streams from youtube, for example: https://www.youtube.com/user/scribblej. i got message:  No playable streams found on this URL
i use streamlink version 0.11.0 and python 3.6.3

...

### Expected / Actual behavior
got message:  No playable streams found on this URL
...

### Reproduction steps / Explicit stream URLs to test

1. ...
2. ...
3. ...

### Logs

_Logs are always required for a bug report, use `-l debug` [(help)](https://streamlink.github.io/cli.html#cmdoption-l)
Make sure to **remove username and password**
You can upload your logs to https://gist.github.com/ or_

```
REPLACE THIS TEXT WITH YOUR LOG
```

### Comments, screenshots, etc.

...

[Love Streamlink? Please consider supporting our collective. Thanks!](https://opencollective.com/streamlink/donate)

**Comment by back-to** [comment]
> i use streamlink version 0.11.0 and python 3.6.3

- check your version `streamlink --version` it's maybe an old version
- run your command with `-l debug`
- **make sure the stream is live** when you try to open it

```
$ streamlink https://www.youtube.com/user/scribblej best -l debug
[cli][debug] OS:         Linux
[cli][debug] Python:     3.6.4
[cli][debug] Streamlink: 0.11.0
[cli][debug] Requests(2.18.4), Socks(1.6.7), Websocket(0.47.0)
[cli][info] Found matching plugin youtube for URL https://www.youtube.com/user/scribblej
[plugin.youtube][debug] get_video_info - 1: Found data
[cli][info] Available streams: audio_mp4, 144p (worst), 240p, 360p, 480p, 720p, 1080p (best)
[cli][info] Opening stream: 1080p (hls)
[stream.hls][debug] Reloading playlist
[stream.hls][debug] First Sequence: 1019831; Last Sequence: 1022710
[stream.hls][debug] Start offset: 0; Duration: None; Start Sequence: 1022708; End Sequence: None
[stream.hls][debug] Adding segment 1022708 to queue
[cli][debug] Pre-buffering 8192 bytes
[stream.hls][debug] Adding segment 1022709 to queue
[stream.hls][debug] Adding segment 1022710 to queue
[stream.hls][debug] Download of segment 1022708 complete
[cli][info] Starting player: /usr/bin/mpv
[stream.hls][debug] Download of segment 1022709 complete
[cli][debug] Writing stream to output
[stream.hls][debug] Download of segment 1022710 complete
[stream.hls][debug] Reloading playlist
[stream.hls][debug] Adding segment 1022711 to queue
[stream.hls][debug] Download of segment 1022711 complete
[stream.hls][debug] Reloading playlist
[stream.hls][debug] Adding segment 1022712 to queue
[stream.hls][debug] Download of segment 1022712 complete
[stream.hls][debug] Reloading playlist
[stream.hls][debug] Adding segment 1022713 to queue
[stream.hls][debug] Download of segment 1022713 complete
[cli][info] Player closed
[stream.hls][debug] Closing worker thread
[stream.hls][debug] Closing writer thread
[cli][info] Stream ended
[cli][info] Closing currently open stream...
```

**Comment by wiemar7** [comment]
i checked version and see this: streamlink-script.py 0.9.0. how it is posible when i unistall previous version and delete streamlink folder?

**Comment by back-to** [comment]
the installpath is different in **0.9.0** _(bug that was fixed in 0.10.0)_

---

you can just delete the old version,

**GUIDE**
https://github.com/streamlink/streamlink/issues/1391#issuecomment-354275115

**Comment by wiemar7** [comment]
yeaaa! i delete and unistall old versions from AppData/Local and Roaming folders and now i got latest version. now i can download this stream :) thank you so much
